README.md

```diff
@@ -3,18 +3,18 @@ An approach to make code "production-ready"
 
 There is no "one true way" to write code or make it production-ready. What _production-ready_ means will highly depend on what the code does and the context in which you will use it. However, it is generally useful to try and prioritise among all the things that you could do to increase the level of confidence that you have in your code, and approach that task in a systematic way.
 
-> The general rule of thumb is: **don't do more than needed**. The sooner your code is in production, the sooner it actually starts being useful. You are only judge of what is _needed_, and it is a difficult question to answer. Different people around you will have different thoughts about what that means. Try to stay open-minded about it, accept that you might not be fully comfortable with the code you ship to production, but be reasonable, think through the consequences and don't put yourself in a position where you'd be tempted to deny responsibility for what you ship — because you _are_ ultimately responsible for what the code you write does.
->
-> I'll do my best to help. Now take a breath, it takes practice, but you can do it! ; ) — [GB](https://github.com/gonzalo-bulnes)
+The general rule of thumb is: **don't do more than needed**. The sooner your code is in production, the sooner it actually starts being useful. You are only judge of what is _needed_, and it is a difficult question to answer. Different people around you will have different thoughts about what that means. Try to stay open-minded about it, accept that you might not be fully comfortable with the code you ship to production, but be reasonable, think through the consequences and don't put yourself in a position where you'd be tempted to deny responsibility for what you ship — because you _are_ ultimately responsible for what the code you write does.
```

Comment:
"AUTHOR'S NOTE" feels a bit too much for me? :grimacing: What about the less formal "NOTE"? Do you think it's enough to be effective @oknono?

```suggestion
> **NOTE**: The general rule of thumb is: **don't do more than needed**. The sooner your code is in production, the sooner it actually starts being useful. You are only judge of what is _needed_, and it is a difficult question to answer. Different people around you will have different thoughts about what that means. Try to stay open-minded about it, accept that you might not be fully comfortable with the code you ship to production, but be reasonable, think through the consequences and don't put yourself in a position where you'd be tempted to deny responsibility for what you ship — because you _are_ ultimately responsible for what the code you write does.
```